vernac/metasyntax.ml

```diff
@@ -1493,7 +1486,7 @@ let recover_notation_syntax ntn =
         synext_notation = ntn;
         synext_notgram = pa_rule }
     with Not_found ->
-      raise NoSyntaxRule in
+      (let entry = match ntn with (InConstrEntry,_) -> "in constr" | (InCustomEntry e,_) -> "in " ^ e in Feedback.msg_info Pp.(str"recover_notation_syntax: not found " ++ str entry); raise NoSyntaxRule) in
```

Comment:
I don't remember. Won't this `msg_info` print something which should be printed only in debug mode?

Reply:
Yes, sorry, I simply forgot to remove it.